package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Apr  3 07:12:34 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Fix initial proposal resetting settings properly before a new
```

Comment:
This is hardly readable/understandable, could you please rephrase it a bit? Keep in mind that this is also read by the customers...